transactions/destroy_treasury.cdc

```diff
@@ -0,0 +1,12 @@
+import DAOTreasury from "../contracts/DAOTreasury.cdc"
+
+transaction() {
+  
+  prepare(signer: AuthAccount) {
+    let treasury <- signer.load<@DAOTreasury.Treasury>(from: DAOTreasury.TreasuryStoragePath)
+    destroy treasury
```

Comment:
can you move `destroy treasury` to the `execute` block?
will need to define `treasury` variable above prepare block and refer to as `self.treasury`

we are trying to be more intentional with the blocks when writing transactions 😄